packages/camltc/camltc.0.9.4/url

```diff
@@ -0,0 +1,2 @@
+git: "https://github.com/toolslive/camltc.git#camltc-0.9.4"
```

Comment:
would it be possible to get a stable archive to merge into opam-repository, rather than a git tag?